providers/flagd/pom.xml

```diff
@@ -11,12 +11,13 @@
     </parent>
     <groupId>dev.openfeature.contrib.providers</groupId>
     <artifactId>flagd</artifactId>
-    <version>0.11.4</version> <!--x-release-please-version -->
+    <version>0.11.23-local</version> <!--x-release-please-version -->
```

Comment:
Revert before merging.

Reply:
Will do, I just want to do some final manual tests